docs/content/7.supported-urls.md

```
@@ -0,0 +1,226 @@
+navigation:
+  icon: i-lucide-link
+---
+
+## Supported URL Formats
+
+Nuxt Lazytube automatically parses and supports various URL formats for both YouTube and Vimeo. You can use any of the formats below - the module will handle the conversion automatically.
+
+## YouTube URLs
+
+### Standard Watch URL
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)